src/bin/ion/commands/mod.rs

```diff
@@ -157,11 +157,6 @@ impl WithIonCliArgument for ClapCommand {
         )
     }
 
-    /// All commands automatically have the "unstable" opt-in flag. This makes it visible.
```

Comment:
🗺️ This impl was moved so the method order would align with the trait's method order.